.evergreen/.evg.yml

```diff
@@ -1069,8 +1069,8 @@ buildvariants:
     - name: "plain-auth-test"
 
 - matrix_name: "aws-auth-test"
-  matrix_spec: { jdk: "jdk11" }
-  display_name: "MONGODB-AWS Auth test"
+  matrix_spec: { auth: "noauth", ssl: "nossl", jdk: "jdk11", version: ["4.4", "latest"], os: "ubuntu" }
```

Comment:
It's a bit confusing that it's an auth test but we have to specify `auth = "noauth"`.  Is is necessary to have that bit, or will it default to `noauth` if not specified?

Reply:
Removed. This was accidental. We actually specify "AUTH: auth" for the AWS tests so this was a noop.